test/endToEndTests.ts

```diff
@@ -40,16 +32,18 @@ describe('satcheljs', () => {
 
     it('mutatorAction dispatches an action and subscribes to it', () => {
         // Arrange
+        const satchel = createTestSatchel();
         let arg1Value;
         let arg2Value;
 
-        let testMutatorAction = mutatorAction('testMutatorAction', function testMutatorAction(
-            arg1: string,
-            arg2: number
-        ) {
-            arg1Value = arg1;
-            arg2Value = arg2;
-        });
+        let testMutatorAction = mutatorAction(
+            satchel,
```

Comment:
I know I mentioned passing this in in the proposal, but seeing this I feel like making it a member method would be better.  What do you think?

`let testMutatorAction = satchel.mutatorAction(...):`

Reply:
Yeah I agree that  having it as a member method would be better as you always need to pass a satchel instance anyways.